frontend/src/app/main/ui/components/context_menu_a11y.scss

```diff
@@ -35,8 +36,12 @@
     border: $s-2 solid var(--panel-border-color);
     background-color: var(--menu-background-color);
     overflow: auto;
+
     & .separator {
-      height: $s-12;
+      border-top: solid 1px var(--color-background-quaternary);
```

Comment:
We don't use "px" units, I think this should be a "$s-1", you can check all the available values in frontend/resources/styles/common/refactor/spacing.scss